src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -420,6 +444,34 @@ private static void close(final Closeable res) {
             this.output = out;
             this.level = lvl;
         }
+
+        /**
+         * Checks if line is part of a stack trace and should be appended.
+         * @param string String to check
+         * @return boolean result
+         */
+        private static boolean shouldAppend(final String string) {
+            final String leftStrip = stripStart(string);
+            return leftStrip.startsWith(PREFIX_AT) || leftStrip.startsWith(PREFIX_CB) || leftStrip.startsWith(PREFIX_DOTS);
+        }
+
+        /**
+         * Strips whitespace at beginning of String
+         * @param string String to strip
+         * @return string Stripped String
+         */
+        private static String stripStart(final String string) {
+            if ((string == null) || string.isEmpty()) {
+                return null;
```

Comment:
@dean-e-clark `null` must be eliminated from the code completely, see http://www.yegor256.com/2014/05/13/why-null-is-bad.html